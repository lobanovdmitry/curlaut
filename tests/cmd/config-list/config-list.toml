bin.name = "curlaut"
args = "config list"
fs.cwd = "fs"

[env.add]
HOME = "."
