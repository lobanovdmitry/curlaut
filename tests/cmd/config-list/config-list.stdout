* Keycloak 'local':
  [ url=http://localhost:8089/, realm=dlobanov, client_id=dlobanov_client, client_secret=, username=user, password=password ]
Keycloak 'local-non-default':
  [ url=http://localhost:8089/, realm=dlobanov, client_id=dlobanov_client, client_secret=, username=user, password=password ]
